src/search/sr_amebsa.c

```
@@ -30,7 +30,7 @@
 #include "sr_simplex.h"
 
 /* The SEARCH driver keeps a global seed. */
-extern long sa_idum;
+extern uint64_t sa_idum;
```

Clang (14.0.6)

[warning] variable 'sa_idum' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)